spec/controllers/users_controller_spec.rb

```diff
@@ -5,15 +5,15 @@
     it 'redirects and displays the flash message if no user is present' do
       delete :destroy
 
-      expect(response).to redirect_to(root_url)
+      expect(response).to redirect_to "http://#{Figaro.env.domain_name}/"
```

Comment:
For some reason, the test now goes to the domain based on `domain_name` instead of `test.host`, so we can't use the URL helpers anymore which apparently only work with `test.host`.